com.unity.perception/Tests/Runtime/GroundTruthTests/SegmentationGroundTruthTests.cs

```diff
@@ -63,6 +63,7 @@ public class SegmentationPassTests : GroundTruthTestBase
     {
         static readonly Color32 k_SemanticPixelValue = new Color32(10, 20, 30, Byte.MaxValue);
         private static readonly Color32 k_InstanceSegmentationPixelValue = new Color32(255,0,0, 255);
+        private static readonly Color32 k_SkyValue = new Color32(10, 20, 30, 40);
```

Comment:
Does the (10, 20, 30, 40) mean something?

Reply:
no, just wanted to use unique values for each channel to be super sure that we get the same data in as out.